respawn_altering/finisher

```diff
@@ -1,5 +1,5 @@
 Finisher- (5)
-- At night, the finisher can choose to inflict further harm on a respawning player, extending their respawn timer by 4 phases.
+- At night, the finisher can choose to inflict further harm on a respawning player, extending their respawn timer by 2 phases.
```

Comment:
the problem with this is that it should be equal to whatever healers is cuz the roles are opposites. so would have to change healer to equal it

Reply:
I only made it unequal because Delaying it by 4 phases sounds like an unpleasant experience, while reducing by 2 phases sounds too less

I am opposed to making it 4 phases for finisher, but I'd not be much sad if we reduced healer to 2 phases, considering they have a buff in respawn time too.

I'd like to know which role you'd prefer changed

Reply:
id prefer healer nerf i think

Reply:
👌 Will do